### PR TITLE
W-17403588-Change-Anypoint-SCM-to-MuleSoft-VCS-CP

### PR DESCRIPTION
**Pull request by Crispy-Salesforce**

### CODE REVIEW

File: modules/ROOT/pages/des-designing-api-specs.adoc [hunk 1]
```diff
@@ -93,7 +93,7 @@ include::anypoint-code-builder::partial$acb-api-design.adoc[tag="apid-best-pract
 include::anypoint-code-builder::partial$acb-reusable-steps.adoc[tag="scm-intro-best-practice"]
 Source control options for API design projects depend on how and when you created the project: 
 
-* As of the February 2024 release, design projects that you create from scratch in Anypoint Code Builder with *Design an API* (see xref:anypoint-code-builder::des-create-api-specs.adoc#design-new-api-spec[Create a New API Specification Project]) or with the command `MuleSoft: Design an API` no longer spawn a corresponding design project in Design Center or use the Anypoint SCM to store and sync changes.  
+* As of the February 2024 release, design projects that you create from scratch in Anypoint Code Builder with *Design an API* (see xref:anypoint-code-builder::des-create-api-specs.adoc#design-new-api-spec[Create a New API Specification Project]) or with the command `MuleSoft: Design an API` no longer spawn a corresponding design project in Design Center or use MuleSoft VCS to store and sync changes.
```
Comment (simoneegeib) [inline modules/ROOT/pages/des-designing-api-specs.adoc]: remove "in Design Center or". The sentence will now look like this:
the command `MuleSoft: Design an API` no longer uses MuleSoft VCS to store and sync changes.

File: modules/ROOT/pages/start-scm.adoc [hunk 2]
```diff
@@ -4,7 +4,7 @@
 :init-local: Initializing and committing your project files into a local repo in the IDE is a prerequisite to publishing and syncing files into a remote repo in GitHub.
 :repo-in-github: If you have an empty repo in GitHub, you can publish your project files to the repo from the IDE.
 :extra-repo: If your project is already configured to sync with a remote repo, you can configure the IDE to publish your project files to a different repo. 
-:extra-repo-continued: This feature is helpful, for example, if you want to use your own repo for an API spec project that you imported from MuleSoft VCS (previously known as Design Center) or if you created your design project from the IDE before the February 2024 release. Both types of projects automatically add a connection to a remote Git repo in the Anypoint SCM.
+:extra-repo-continued: This feature is helpful, for example, if you want to use your own repo for an API spec project that you imported from MuleSoft VCS (previously known as Design Center) or if you created your design project from the IDE before the February 2024 release. Both types of projects automatically add a connection to a remote Git repo in MuleSoft VCS.
```
Comment (simoneegeib) [inline modules/ROOT/pages/start-scm.adoc]: remove "(previously known as Design Center)"

File: modules/ROOT/pages/_partials/acb-api-design.adoc [hunk 3]
```diff
@@ -22,7 +22,7 @@ API design projects xref:anypoint-code-builder::des-create-api-specs.adoc#import
 
 //
 // tag::note-asyncapi[]
-If your AsyncAPI specification is not published to Exchange, see xref:design-center::design-publish.adoc[]. To create an AsyncAPI specification before publishing to Exchange, see xref:design-center::design-async-api.adoc[] in the Design Center documentation. 
+If your AsyncAPI specification isn't published to Exchange, see xref:design-center::design-publish.adoc[]. To create an AsyncAPI specification before publishing to Exchange, see xref:design-center::design-async-api.adoc[] in the Design Center documentation.
```
Comment (simoneegeib) [inline modules/ROOT/pages/_partials/acb-api-design.adoc]: This should link to the ACB docs that talk about designing an AsyncAPI

File: modules/ROOT/pages/des-delete-api-projects.adoc [hunk 4]
```diff
@@ -3,7 +3,15 @@
...
 
-To delete a design project for your API spec or fragment from Anypoint Code Builder, delete the entire project folder. If you imported the API spec from MuleSoft VCS or if your project was created with earlier versions of Anypoint Code Builder (prior to February 2024) that required storage of the spec in the Anypoint SCM, you can also delete the project from MuleSoft VCS by accessing Design Center. Deletion is necessary if you want to reuse the project name for a new design project in Anypoint Code Builder. 
+Deleting API specs and fragments varies depending on the project creation date and project hosting.
+
+* To delete a design project hosted locally on Anypoint Code Builder, delete the entire project folder. 
+* To delete a design project hosted on US cloud or EU cloud with an API spec imported from MuleSoft VCS, you must delete the project from MuleSoft VCS by accessing Design Center.
+* To delete a design project project created with versions of Anypoint Code Builder released prior to February 2024 that required storage of the spec in the MuleSoft VCS, you must delete the project from MuleSoft VCS by accessing Design Center.
```
Comment (simoneegeib) [inline modules/ROOT/pages/des-delete-api-projects.adoc]: there's a typo: project is mentioned twice

File: modules/ROOT/pages/_partials/acb-api-design.adoc (hunk 3 above)
Comment (Crispy-Salesforce) [reply modules/ROOT/pages/_partials/acb-api-design.adoc]: OK! Thanks!

File: modules/ROOT/pages/des-delete-api-projects.adoc (hunk 4 above)
Comment (Crispy-Salesforce) [reply modules/ROOT/pages/des-delete-api-projects.adoc]: Great catch! Thanks!

File: modules/ROOT/pages/des-designing-api-specs.adoc (hunk 1 above)
Comment (Crispy-Salesforce) [reply modules/ROOT/pages/des-designing-api-specs.adoc]: OK! Thanks!

File: modules/ROOT/pages/start-scm.adoc (hunk 2 above)
Comment (Crispy-Salesforce) [reply modules/ROOT/pages/start-scm.adoc]: OK! Thanks!

**Review by simoneegeib — APPROVED** [review]

File: modules/ROOT/pages/des-delete-api-projects.adoc [hunk 5]
```diff
@@ -3,7 +3,15 @@
 
 include::reuse::partial$beta-banner.adoc[tag="anypoint-code-builder"]
 
-To delete a design project for your API spec or fragment from Anypoint Code Builder, delete the entire project folder. If you imported the API spec from MuleSoft VCS or if your project was created with earlier versions of Anypoint Code Builder (prior to February 2024) that required storage of the spec in the Anypoint SCM, you can also delete the project from MuleSoft VCS by accessing Design Center. Deletion is necessary if you want to reuse the project name for a new design project in Anypoint Code Builder. 
+Deleting API specs and fragments varies depending on the project creation date and project hosting.
+
+* To delete a design project hosted locally on Anypoint Code Builder, delete the entire project folder. 
+* To delete a design project hosted on US cloud or EU cloud with an API spec imported from MuleSoft VCS, you must delete the project from MuleSoft VCS by accessing Design Center.
```
Comment (dukesphere) [inline modules/ROOT/pages/des-delete-api-projects.adoc:9]: what about our new clouds (Japan and Canada)?

**Review by dukesphere — APPROVED** [review]
approved with a question

File: modules/ROOT/pages/des-delete-api-projects.adoc (hunk 5 above)
Comment (Crispy-Salesforce) [reply modules/ROOT/pages/des-delete-api-projects.adoc:9]: Explanation available on line 54+. Thanks!

**Pull request merged**